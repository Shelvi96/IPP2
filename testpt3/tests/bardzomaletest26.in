NEW ID

13016862:5>:::22
511;4001;4>;1844152
05097:0542>144
0451>2;:3;2;0
52;2>9:497
3:572>7:419
47139>8892808
24618131>90
7>142537143
14>94;6:744
574658>4481:6
571>825;312
@9584230
@098958242611;2117040
@5472035
@460569:6;
@1:829:439879752276642
@23700574082823::6;;60
@;60:1:3
@09:;:;9979919;4;23259238936:
@15:34:1
@7:28735:
@842:9;04;10:;6394612701929896;
@2
@3456916:182;;;464:5443
