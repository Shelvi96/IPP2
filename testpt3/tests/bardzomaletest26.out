0
1100011
0
0
1031001
10010
0
1393214257741824
0
0
50234021784316139
0
10111100
